daemon/daemon.go

```diff
@@ -19,6 +19,8 @@ import (
 	protocol2 "gopkg.in/bblfsh/sdk.v2/protocol"
 )
 
+const BuildDateFormat = "2006-01-02T15:04:05-0700"
```

Comment:
This is not really related to the daemon package. It's only the `bblfshd/main.go` who parses it that way.

Reply:
It's just retrofitting existing existing code structure to minimize the changes - if you look below, this is used in `daemon_test.go`, thus the location.

Reply:
But the test is free to use any format. `NewDaemon` doesn't enforce any specific format.

Reply:
sure, but usually it's kind of good idea to test with the format that is going to be used in prod, no?

Reply:
The point is that we should either move the constant to this package and dictate the format there (that is not a good idea I think) or somehow write the test for the binary itself (`package main`).

Reply:
Got it! Decouple date format parsing and daemon package tests in d4ea40b